src/tools/perf/perftest.h

```diff
@@ -33,8 +33,9 @@ enum {
 };
 
 typedef struct sock_rte_group {
+    int                          connfds[2]; /* "0" - for send, "1" - for recv */
```

Comment:
IMO using 2 variables (sendfd, recvfd) would be more clear than an array

Reply:
socketpair func requires array

Reply:
can use temp array?